packages/starknet-snap/src/utils/rpc-provider.test.ts

```diff
@@ -0,0 +1,25 @@
+import { constants } from 'starknet';
+
+import { Config } from '../config';
+import { getRPCUrl } from './rpc-provider';
+
+describe('getRPCUrl', () => {
+  Config.rpcApiKey = 'test';
```

Comment:
```suggestion
  beforeEach(function () {
    Config.rpcApiKey = 'test';
  });
  afterEach(function () {
    Config.rpcApiKey = undefined
  });
```

reset it, in case you have some unexpected behaviour (the way using before and after is not suggest by MM, but i think it is okay for that case)